app/controllers/api/v3/timelines/for_you_controller.rb

```diff
@@ -109,7 +110,7 @@ def personalized_for_you_list_statuses
   end
 
   def personalzied_feed
-    ForYouFeed.new('personal', @account.id)
+    ForYouFeed.new('personal', @user[:acct])
   end
```

Comment:
account.id is tied to the moth.social instance. using the user's full username for cache key decouples it from having to do any moth.social specific lookup when read/writing to it. This is pervious logic from original FY